### ISSUE TITLE
Gadgetron fails to compile with self-built Boost

### ISSUE THREAD

**Issue by KrisThielemans** [SyneRBI/SIRF-SuperBuild#662]
https://github.com/SyneRBI/SIRF-SuperBuild/runs/5435576973?check_suite_focus=true#step:7:25637
```
[ 33%] Linking CXX shared library libgadgetron_toolbox_fatwater.so
/usr/bin/ld: CMakeFiles/gadgetron_toolbox_fatwater.dir/correct_frequency_shift.cpp.o: relocation R_X86_64_PC32 against undefined hidden symbol `_ZTCN5boost10wrapexceptIN9Gadgetron9bad_allocEEE8_S2_' can not be used when making a shared object
/usr/bin/ld: final link failed: bad value
```

More than likely unrelated is https://github.com/llvm/llvm-project/issues/39830, as this is `clang`, but the above is with `gcc` 9.

@paskino any ideas?

**Comment by paskino** [comment]
Does this happen with the new version of Boost after #656?

**Comment by KrisThielemans** [comment]
yes. this is on latest `master`.
https://github.com/SyneRBI/SIRF-SuperBuild/runs/5435576973?check_suite_focus=true#step:7:12
shows it's Boost 1.78.0

**Comment by paskino** [comment]
I'm slowly trying to process all this in #663

**Comment by paskino** [comment]
Does #663 close this issue?

**Comment by KrisThielemans** [comment]
not sure. do we still have failures? I guess not, so in that case I'd close this.

**Comment by paskino** [comment]
I just merged #663 and it was all green, so I'd close.
